CMakeLists.txt

```diff
@@ -410,6 +424,11 @@ set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} /STACK:1572864")
 # Debug build specific flags
 set(CMAKE_SHARED_LINKER_FLAGS_DEBUG "/NOVCFEATURE")
 
+# Checked build specific flags
+set(CMAKE_SHARED_LINKER_FLAGS_CHECKED "${CMAKE_SHARED_LINKER_FLAGS_CHECKED} /OPT:REF /OPT:NOICF /NOVCFEATURE")
```

Comment:
@mmitche The only info on /NOVCFEATURE that I found is "/NOVCFEATURE forces linker to emit old .pdb format. It is required for scan.exe tool to work". Looks like scan.exe is a static contract analyzer. Maybe we are not running it on release binaries? Let me know if you know more about /NOVCFEATURE or scan.exe or think that /NOVCFEATURE should be passed when linking release binaries. For checked binaries I just matched we do for debug binaries.